### ISSUE TITLE
Plugin no longer working after upgrading to Smart Hub Pro

### ISSUE THREAD

**Issue by johnyoungberg** [balansse/homebridge-vivint#137]
I am testing a new panel version and the plugin appears to not be able to recognize it. This is the error I am seeing in the logs

```
Error while bootstrapping accessories: TypeError: Cannot read properties of undefined (reading 'PanelId')
    at Function.VivintApiModule.VivintApi.login (/usr/lib/node_modules/@balansse/homebridge-vivint/lib/vivint_api.js:210:47)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by quentinlindsey** [comment]
I'm getting the exact same error.

**Comment by johnyoungberg** [comment]
Is your system also using the Smart Hub Pro?

**Comment by azimjs** [comment]
@johnyoungberg were you able to figure this one out ? I think I have the Vivint Smart hub as well and getting the exact same error.

**Comment by johnyoungberg** [comment]
No, I stopped using the plugin. I check back from time to time to see if it is fixed but after MyQ breaking and this issue I’m realizing I don’t have much interest in building my smart home ecosystem around hobby projects.
No hate to any of the developers who have put time into this or other plugins. Their efforts are selfless and I think that is rad.

**Comment by azimjs** [comment]
Ah ! Sorry to hear that ! 
I did some further debugging last night, and while I’m not sure if this applies to your case, the error I encountered was due to Vivint’s API returning an empty array of `System`, instead sending an array of `RestrictedSystem`.
I did make some updates, and the plugin was able to retrieve the PanelId. However, I then encountered this error:
```
Error while bootstrapping accessories: StatusCodeError: 403 - "{\"code\": 403, \"msg\": \"Attempted to access an unauthorized Smarthome System\", \"msg_id\": null}"
```

This kinda makes sense in my case because, I don't have an active Vivint subscription. I was trying to see if Homebridge could work without a subscription (bought the house with Vivint devices already installed and paid off)

**Comment by johnyoungberg** [comment]
This is resolved.
The issue was on my end. After upgrading the panel, the user I created for Homebridge needed to be re-invited to the home. After doing that all errors are fixed.
